lib/cfonb/operation_detail/mmo.rb

```diff
@@ -14,6 +14,7 @@ def self.apply(operation, line)
         sign = operation.amount <=> 0 # the detail amount is unsigned
 
         operation.original_amount = sign * BigDecimal(line.detail[4..17]) / (10**scale)
+        operation.exchange_rate = BigDecimal(line.detail[-4..-1]) / 1000
```

Comment:
For reference the official definition doe snot state it will always be the exchange rate
![image](https://github.com/pennylane-hq/cfonb/assets/32439295/486cd83c-5105-4be2-b94e-ab4990f2b4e0)